pkg/analyzer/java/parse_manifest_test.go

```diff
@@ -0,0 +1,49 @@
+package java
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+const testManifestMF = `
```

Comment:
@RTann 
Maybe we should add this specific example to the Claircore to manifest parsing. What do you think?

Reply:
ClairCore doesn't have this problem, as they only read the "main" section and also stop at `Name`: https://github.com/quay/claircore/blob/main/java/jar/jar.go#L526